src/Agent.Listener/Configuration/AutoLogonRegistryManager.cs

```diff
@@ -286,7 +286,7 @@ private void UpdateUserSpecificRegistrySettings(CommandSettings command, string
 
         private void UpdateScreenSaverSettings(CommandSettings command, string securityId)
         {
-            if(!command.GetDisableScreenSaver())
+            if(command.GetKeepScreenSaver())
```

Comment:
> we just want to keep it separate. It is different from service usage, even though screensaver is not disabled as UI testing can be done.

I don't understand. What I'm trying to understand is: Can UI testing be done when the screen saver is on?